### DISCUSSION TITLE
issue setting screen resolution

### DISCUSSION THREAD

**Discussion by shlandrews** [bareboat-necessities/lysmarine_gen#405] [General]
I have a 1920x1200 16:10 60hz touch screen, in openplotter if I just set the correct resolution in screen configuration tool it works fine. However in BBN, when I do this it works for a few seconds then switches back to 1920x1080 resolution. I have tried the following in config.txt to no success -
changed this from 1 to 0 to:
display_auto_detect=0
added the following:
hdmi_group=2
hdmi_mode=87
hdmi_cvt 1920 1200 60 3 0 0 0

reboot and goes to default 1920x1080 and same issue still if try use the screen configuration tool. I wonder if I have just missed a setting in the config.txt file as I noted, this monitor works fine at correct resolution with openplotterv3 using the screen config tool.

**Comment by mgrouch** [comment]
Latest BBN is bookworm based so it uses 
kms driver instead of fkms. 
the way you were trying to set resolution is for fkms driver. Kms driver settings need to be done in /boot/cmdline.txt

**Comment by mgrouch** [comment]
https://bareboat-necessities.github.io/my-bareboat/bareboat-os.html#_setting_up_monitor

part about kms driver on bookworm doesn’t help?

**Reply by shlandrews** [reply]
I don't know how I missed that, but sometimes you do - thanks, resolved
